suite/tests/api/ir_regdeps.c

```diff
@@ -328,6 +440,22 @@ test_instr_create_encode_decode_synthetic_riscv64(void *dc)
 }
 #endif
 
+void
+test_virtual_register(void)
+{
+    const char rv0[] = "rv0";
+    ASSERT(strncmp(get_virtual_register_name((reg_id_t)0), rv0, sizeof(rv0)) == 0);
```

Comment:
Shouldn't these use the enum values?

Reply:
Now that DR_REG_V are public, yes. Done.